app/pages/lab/media.jsx

```diff
@@ -36,7 +36,7 @@ export default class EditMediaPage extends React.Component {
 
 EditMediaPage.defaultProps = {
   project: {},
-  validSubjectExtensions: ['.jpg', '.jpeg', '.png', '.gif', '.svg'],
+  validSubjectExtensions: ['.jpg', '.jpeg', '.png', '.gif', '.svg', '.mp3'],
```

Comment:
I believe this should be updated to include video extensions?

Reply:
Yep. At the moment Panoptes blocks video uploads, so I've not let project owners know that that option exists.